app/Rest/Employees_Controller.php

```diff
@@ -0,0 +1,475 @@
+			]
+		);
+	}
+
+	/**
+	 * Get_item
+	 *
+	 * @param \WP_REST_Request $request Full details about the request.
+	 *
+	 * @return \WP_REST_Response
+	 */
+	public function delete_item( $request ) {
+		$request->set_param( 'context', 'edit' );
+
+		$result = wp_delete_post( $request['id'], true );
+
+		if ( is_wp_error( $result ) ) {
+			return $result;
+		}
+
+		return $this->response(
+			[
+				'status' => 'Employee has been deleted',
+				'id'     => $request['id'],
+			]
+		);
+	}
+
+	/**
+	 * Item_permissions_check
+	 *
+	 * @param \WP_REST_Request $request Full details about the request.
+	 *
+	 * @return bool|\WP_Error True if the request has read access for the item, WP_Error object otherwise.
+	 */
+	public function item_permissions_check( $request ) {
+		$post = $this->get_employee( $request['id'] );
+		if ( is_wp_error( $post ) ) {
+			return $post;
+		}
+
+		return true;
+	}
+
+	/**
+	 * Get_collection_params
+	 *
+	 * @return array
+	 */
+	public function get_collection_params() {
+		$query_params = parent::get_collection_params();
+
+		$query_params['context']['default'] = 'view';
+
+		$query_params['exclude'] = array(
+			'description' => __( 'Ensure result set excludes specific IDs.' ),
+			'type'        => 'array',
+			'items'       => array(
+				'type' => 'integer',
+			),
+			'default'     => array(),
+		);
+
+		$query_params['include'] = array(
+			'description' => __( 'Limit result set to specific IDs.' ),
+			'type'        => 'array',
+			'items'       => array(
+				'type' => 'integer',
+			),
+			'default'     => array(),
+		);
+
+		$query_params['offset'] = array(
+			'description' => __( 'Offset the result set by a specific number of items.' ),
+			'type'        => 'integer',
+		);
+
+		$query_params['order'] = array(
+			'description' => __( 'Order sort attribute ascending or descending.' ),
+			'type'        => 'string',
+			'default'     => 'desc',
+			'enum'        => array( 'asc', 'desc' ),
+		);
+
+		$query_params['orderby'] = array(
+			'description' => __( 'Sort collection by object attribute.' ),
+			'type'        => 'string',
+			'default'     => 'date',
+			'enum'        => array(
+				'author',
+				'date',
+				'id',
+				'include',
+				'modified',
+				'parent',
+				'relevance',
+				'slug',
+				'include_slugs',
+				'title',
+			),
+		);
+
+		return $query_params;
+	}
+
+	/**
+	 * Get_item_schema
+	 *
+	 * @return array
+	 */
+	public function get_item_schema() {
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => Employee_Post_Type::$ID,
+			'type'       => 'object',
+			'properties' => array(
+				'title'     => array(
+					'description' => sprintf( __( 'The title for the %s.' ), Employee_Post_Type::$ID ),
+					'type'        => 'string',
+				),
+				'content'   => array(
+					'description' => sprintf( __( 'The content for the %s.' ), Employee_Post_Type::$ID ),
+					'type'        => 'string',
+				),
+				'acf_field' => array(
```

Comment:
what is this? Our theme has specific examples for employee fields (like position, bio, etc) - use these fields, not some temporary name.